src/spid-sdk.js

```diff
@@ -27,6 +28,15 @@ function init(opts, callback) {
     }
 }
 
+function tryVarnishCookie(session) {
```

Comment:
I would like this method moved. The SDK file is otherwise pretty clean, just containing methods of the public interface of the sdk, the API. This is an internal method.

I'm trying to figure out a better location, and maybe you can help me out? Put in cookie module perhaps? What do you think?